### PR TITLE
circleci conf: require DOCKER_DEPLOY=true to do any dockerhubby things

### PR DESCRIPTION
**Pull request by milescrabill**
@willkg r?

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -72,5 +72,5 @@ jobs:
               bin/ci/deploy-dockerhub.sh "circle_build_$CIRCLE_BUILD_NUM"
             fi
             if [ -n "${CIRCLE_TAG}"]; then
```
Comment (willkg) [inline .circleci/config.yml]: I'm pretty sure you need a space between `"` and `]` at the end of this test, too.

Comment (willkg) [inline .circleci/config.yml]: Shame you can't run shellcheck on this circle config file. Maybe this is complex enough it's worth moving to a shell script?

File: bin/ci/deploy-dockerhub.sh [hunk 2]
```diff
@@ -22,17 +22,19 @@ function retry() {
     return 0
 }
 
-# configure docker creds
-retry 3 docker login -e="$DOCKER_EMAIL" -u="$DOCKER_USERNAME" -p="$DOCKER_PASSWORD"
+if [[ "$DOCKER_DEPLOY" == true ]]; then
```
Comment (willkg) [inline bin/ci/deploy-dockerhub.sh]: The Antenna version doesn't have this. Is it good to keep these files in sync between projects?

Comment (milescrabill) [inline bin/ci/deploy-dockerhub.sh]: Yes - definitely a good idea. I'll put in a PR now.

**Pull request merged**